index.html

```diff
@@ -918,107 +918,11 @@ <h2 id="BaseAudioContext">
-              <li>
-                <a href="#queue">Queue a control message</a> to suspend the
-                <a>AudioContext</a>.
-              </li>
-              <li>Return <em>promise</em>.
-              </li>
-            </ol>
-            <p>
-              Running a <a>control message</a> to suspend an
-              <a>AudioContext</a> means running these steps on the <a>rendering
-              thread</a>:
-            </p>
-            <ol>
-              <li>Attempt to <a>release system resources</a>.
-              </li>
-              <li>Set the <a>rendering thread state</a> on the
-              <a>AudioContext</a> to <code>suspended</code>.
-              </li>
-              <li>Queue a task on the <a>control thread</a>'s event loop, to
-              execute these steps:
-                <ol>
-                  <li>Resolve <em>promise</em>.
-                  </li>
-                  <li>If the <a href="#widl-audiocontext-state">state</a>
-                  attribute of the <a>AudioContext</a> is not already
-                  <code>suspended</code>:
-                    <ol>
-                      <li>Set the <a href="#widl-audiocontext-state">state</a>
-                      attribute of the <a>AudioContext</a> to
-                      <code>suspended</code>.
-                      </li>
-                      <li>Queue a task to fire a simple event named
-                      <code>statechange</code> at the <a>AudioContext</a>.
-                      </li>
-                    </ol>
-                  </li>
-                </ol>
-              </li>
-            </ol>
-            <p>
-              While a <a>BaseAudioContext</a> is suspended,
-              <code>MediaStream</code>s will have their output ignored; that
-              is, data will be lost by the real time nature of media streams.
-              <code>HTMLMediaElement</code>s will similarly have their output
-              ignored until the system is resumed.
-              <a>AudioWorkletProcessor</a>s and <a>ScriptProcessorNode</a>s
-              will cease to have their processing handlers invoked while
-              suspended, but will resume when resumed. For the purpose of
-              <a>AnalyserNode</a> window functions, the data is considered as a
-              continuous stream - i.e. the
-              <code>resume()</code>/<code>suspend()</code> does not cause
-              silence to appear in the <a>AnalyserNode</a>'s stream of data. In
-              particular, calling <a>AnalyserNode</a> functions repeatedly when
-              a <a>BaseAudioContext</a> is suspended MUST return the same data.
-            </p>
-          </dd>
-          <dt>
             Promise&lt;void&gt; resume()
           </dt>
           <dd>
             <p>
-              Resumes the progression of the
-              <a><code>AudioContext</code></a>'s' <a href=
+              Resumes the progression of the <a>BaseAudioContext</a>'s <a href=
```

Comment:
2. The resume() method still belongs to `BaseAudioContext`: so the original definition stays in the same place, but I had to change all the instances of `AudioContext` to `BaseAudioContext`.